core/domain/suggestion_services.py

```diff
@@ -588,24 +588,24 @@ def get_all_suggestions_that_can_be_reviewed_by_user(user_id):
     ])
 
 
-def get_reviewable_suggestions_for_translate_content(
+def get_reviewable_translation_suggestions(
         user_id, opportunity_summary_exp_ids):
-    """Returns a list of suggestions of type translate_content
-     matching the passed opportunity ids which the user can review.
+    """Returns a list of translation suggestions matching the
+     passed opportunity IDs which the user can review.
 
     Args:
         user_id: str. The ID of the user.
-        opportunity_summary_exp_ids: list(str) or None.
+        opportunity_summary_exp_ids: list(str) or NONE.
```

Comment:
None, not NONE

Reply:
Oh, sorry, I thought when Sagang said "captilize: none" he meant it completely. Looks like the earlier version was right, but I made it all lower in one of the arg descriptions.